### DISCUSSION TITLE
Choice of a PC for CARLA

### DISCUSSION THREAD

**Discussion by sebastiennadler** [carla-simulator/carla#5646] [General]
Hi everybody, especially CARLA's developper. 

I am wondering what would be the best choice to use CARLA simulator for a small laboratory (with a limited budget). I'am wondering if this configuration Memory PC Intel Core i5-12400F 6X 4.4 GHz, 16 Go DDR4 RAM, 1TB M.2 SSD, AMD Radeon RX 6800 XT 16GB, Windows 11 Pro (https://amzn.eu/d/eBW0lkw) would be optimal ? 

Can I have an advice of a specialist to choose a hardware to run CARLA ? 

Thank you so much, 

Sébastien Nadler
R&D projects Leader
ASTEK

**Comment by RaviBeagle** [comment]
I am not a specialist, only a beginner, but from my usage experience I can do a comparison. 
I think primarily its the memory size and capability of the GPU that matters than CPU and CPU memory.

My GPU is:

0  NVIDIA RTX A3000...  Off  | 00000000:01:00.0  On |                  N/A |
| N/A   47C    P8    15W /  N/A |    722MiB /  6144MiB |      8%      Default

https://technical.city/en/video/Radeon-RX-6800-vs-RTX-A3000-mobile

NVIDIA RTX A3000  has 6GB GDDR memory and I am able to run CARLA 0.9.12 with EPIC settings. Of course the laptop fan is running all the time and it gets heated up a lot. A desktop with proper cooling should work better.
